extensions/amp-list/0.1/amp-list.js

```diff
@@ -492,6 +492,11 @@ export class AmpList extends AMP.BaseElement {
       this.toggleLoading(true, /* opt_force */ true);
       this.mutateElement(() => {
         this.toggleFallback_(false);
+        // Clean up bindings in children before removing them from DOM.
```

Comment:
do we need to consider the other places where we remove items? 

e.g.
https://github.com/ampproject/amphtml/blob/master/extensions/amp-list/0.1/amp-list.js#L784
https://github.com/ampproject/amphtml/blob/master/extensions/amp-list/0.1/amp-list.js#L711

Reply:
Good thinking -- I already checked. The first case is already covered. The second case is unrelated.